lib/system/assertions.nim

```diff
@@ -0,0 +1,89 @@
+include "system/helpers"
+
+when not declared(sysFatal):
```

Comment:
Why is this ``when`` required?

Reply:
That's the way it is [currently done in system.nim](https://github.com/nim-lang/Nim/blob/devel/lib/system.nim#L3039) so I kept it the same.